source/_components/binary_sensor.template.markdown

```diff
@@ -217,45 +228,17 @@ binary_sensor:
              or is_state('binary_sensor.porch_ms6_1_129', 'on')
              or is_state('binary_sensor.family_room_144', 'on') }}
 ```
-{% endraw %}
-
-### Device Tracker sensor with Latitude and Longitude Attributes
```

Comment:
Why is this being removed? Seems unrelated.

Reply:
Sme as above. The 'attribute_attribute' feature was added after my original pull request 4 weeks ago.